src/rules/indentation.js

```diff
@@ -1,12 +1,12 @@
 var _ = require('lodash');
 var rule = 'indentation';
 
-var availableConfigs = {
```

Comment:
I would rather fix this in rule js where the actual error is (I have inverted the options of the ternary operator)

This is the diff of the fix.
```javascript
-    var extraConfig = typeof(ruleConfig[1]) === String ? ruleConfig[1] : [ruleConfig[1]];
-
+    var extraConfig = typeof(ruleConfig[1]) === 'string' ? [ruleConfig[1]] : ruleConfig[1]; 
```